CMakeLists.txt

```diff
@@ -188,28 +188,28 @@ target_include_directories(onnx PUBLIC ${ONNX_ROOT} "${CMAKE_CURRENT_BINARY_DIR}
 target_link_libraries(onnx PUBLIC onnx_proto)
 
 if(BUILD_PYTHON)
-  add_library(onnx_pybind MODULE "${ONNX_ROOT}/onnx/cpp2py_export.cc")
-  target_include_directories(onnx_pybind PRIVATE "${CMAKE_CURRENT_BINARY_DIR}" "${PROTOBUF_INCLUDE_DIRS}" "${PYTHON_INCLUDE_DIR}")
+  add_library(onnx_cpp2py_export SHARED "${ONNX_ROOT}/onnx/cpp2py_export.cc")
```

Comment:
`MODULE` is the recommended type as it's a Python plugin. https://cmake.org/cmake/help/v3.0/command/add_library.html

Reply:
++ meant to change it back